.changes/next-release/enhancement-configprovider-27540.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "enhancement",
+  "category": "configprovider",
+  "description": "Use shallow copy instead of deep copy for config value store"
```

Comment:
Let's rephrase this to be more specific on where we are introducing the shallow copies e.g. `Always use shallow copy of session config value store for clients. This ensures that updates to the session's config value store cannot change client configuration after client instantiation.`

Reply:
Agree for more details, but is that completely accurate? Wouldn't it be:

> This ensures that updates to a config value store used for client instantiation cannot change the session's config value store or subsequent client instantiations.

Reply:
That is true as well but that would seem to indicate the change was more to fix a bug. I'm also fine with just removing the second sentence and leaving it with the first one